### PR TITLE
feat(compute): add compute consistency group clone sample

### PR DESCRIPTION
**Pull request by TetyanaYahodska**
## Description
Documentation - https://cloud.google.com/compute/docs/disks/async-pd/manage-async-disks?authuser=0#clone_all_disks_in_a_consistency_group
Fixes #<ISSUE-NUMBER>

Note: Before submitting a pull request, please open an issue for discussion if you are not associated with Google.

## Checklist

- [x] I have followed [Sample Format Guide](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/SAMPLE_FORMAT.md)
- [ ] `pom.xml` parent set to latest `shared-configuration`
- [ ] Appropriate changes to README are included in PR
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [x] **Tests** pass:   `mvn clean verify` **required**
- [x] **Lint**  passes: `mvn -P lint checkstyle:check` **required**
- [ ] **Static Analysis**:  `mvn -P lint clean compile pmd:cpd-check spotbugs:check` **advisory only**
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/.github/CODEOWNERS) with the codeowners for this sample
- [ ] This sample adds a new **Product API**, and I updated the [Blunderbuss issue/PR auto-assigner](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/.github/blunderbuss.yml) with the codeowners for this sample 
- [ ] Please **merge** this PR for me once it is approved

### CODE REVIEW

File: compute/cloud-client/src/main/java/compute/disks/consistencygroup/CloneRegionalDisksFromConsistencyGroup.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+    cloneRegionalDisksFromConsistencyGroup(project, region, consistencyGroupName);
+  }
+
+  // Clones regional disks from a consistency group.
+  public static Status cloneRegionalDisksFromConsistencyGroup(
+          String project, String region, String consistencyGroupName)
+          throws IOException, InterruptedException, ExecutionException, TimeoutException {
+    String sourceConsistencyGroupPolicy = String.format(
+            "projects/%s/regions/%s/resourcePolicies/%s", project, region, consistencyGroupName);
```
Comment (Sita04) [inline compute/cloud-client/src/main/java/compute/disks/consistencygroup/CloneRegionalDisksFromConsistencyGroup.java:50]: The project, region here are different from the ones in line 56 and 57. 

This refers to the consistency group project and region, whereas line 56 and 57 refers to where the disks should be cloned in.

Kindly modify the samples to get these extra inputs and make changes accordingly.

**Review by Sita04 — CHANGES_REQUESTED** [review]

File: compute/cloud-client/src/main/java/compute/disks/consistencygroup/CloneRegionalDisksFromConsistencyGroup.java (hunk 1 above)
Comment (TetyanaYahodska) [reply compute/cloud-client/src/main/java/compute/disks/consistencygroup/CloneRegionalDisksFromConsistencyGroup.java:50]: There is no mistake. The disk region and consistency group must be the same. Here is a merged sample in Python-  https://github.com/GoogleCloudPlatform/python-docs-samples/blob/9531bf469545820624a2ef0fa1a2aaa965dc2c99/compute/client_library/snippets/disks/%D1%81onsistency_groups/clone_disks_consistency_group.py#L22

File: compute/cloud-client/src/main/java/compute/disks/consistencygroup/CloneZonalDisksFromConsistencyGroup.java [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+import java.util.concurrent.TimeoutException;
+
+public class CloneZonalDisksFromConsistencyGroup {
+  public static void main(String[] args)
+          throws IOException, ExecutionException, InterruptedException, TimeoutException {
+    // TODO(developer): Replace these variables before running the sample.
+    // Project ID or project number of the Cloud project you want to use.
+    String project = "YOUR_PROJECT_ID";
+    // Zone in which your disks are located.
+    String disksLocation = "us-central1-a";
```
Comment (Sita04) [inline compute/cloud-client/src/main/java/compute/disks/consistencygroup/CloneZonalDisksFromConsistencyGroup.java:37]: Modify the parameter name to zone.

**Review by Sita04 — APPROVED** [review]
Kindly address the comment before merging.

File: compute/cloud-client/src/main/java/compute/disks/consistencygroup/CloneZonalDisksFromConsistencyGroup.java (hunk 2 above)
Comment (TetyanaYahodska) [reply compute/cloud-client/src/main/java/compute/disks/consistencygroup/CloneZonalDisksFromConsistencyGroup.java]: Thank you! Fixed naming

**Pull request merged**